src/Service/S3/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@
 - Added `me-central-1` region
 - Added `ap-southeast-3` region
 - AWS api-change: Added `eu-central-2` region
+- AWS api-change: S3 on Outposts launches support for automatic bucket-style alias. You can use the automatic access point alias instead of an access point ARN for any object-level operation in an Outposts bucket.
```

Comment:
None of these change log entries are relevant, and we're missing a note for the new eu region. Probably we should consolidate the changelog entries for each new region to one entry.